lib/ably/rest/client.rb

```diff
@@ -357,20 +362,19 @@ def delete(path, params, options = {})
         raw_request(:delete, path, params, options)
       end
 
-      # Perform an HTTP request to the Ably API
-      # This is a convenience for customers who wish to use bleeding edge REST API functionality
-      # that is either not documented or is not included in the API for our client libraries.
-      # The REST client library provides a function to issue HTTP requests to the Ably endpoints
-      # with all the built in functionality of the library such as authentication, paging,
-      # fallback hosts, MsgPack and JSON support etc.
+      # Makes a REST request to a provided path. This is provided as a convenience for developers who wish to use REST API
+      # functionality that is either not documented or is not yet included in the public API, without having to directly
+      # handle features such as authentication, paging, fallback hosts, MsgPack and JSON support.
+      #
+      # @spec RSC19
       #
-      # @param method  [Symbol]    The HTTP method symbol such as +:get+, +:post+, +:put+
-      # @param path    [String]    The path of the URL such +/channel/foo/publish+
-      # @param params  [Hash, nil] Optional querystring params
-      # @param body    [Hash, nil] Optional body for the POST or PUT request, must be nil or a JSON-like object
-      # @param headers [Hash, nil] Optional additional headers
+      # @param method  [Symbol]    The request method to use, such as GET, POST.
```

Comment:
I think we should keep the examples here as the actual symbols - `:get`, `:post`, etc

Reply:
Fixed